diff --git a/node_modules/@react-native-firebase/firestore/RNFBFirestore.podspec b/node_modules/@react-native-firebase/firestore/RNFBFirestore.podspec
index 861e763..cfdcb01 100644
--- a/node_modules/@react-native-firebase/firestore/RNFBFirestore.podspec
+++ b/node_modules/@react-native-firebase/firestore/RNFBFirestore.podspec
@@ -41,7 +41,7 @@ Pod::Spec.new do |s|
 
   # required until firestore-ios-sdk-frameworks is updated, otherwise users of that distribution will have compile failures
   # see https://github.com/invertase/firestore-ios-sdk-frameworks/issues/59
-  s.dependency          'nanopb', '>= 2.30908.0', '< 2.30910.0'
+  # s.dependency          'nanopb', '>= 2.30908.0', '< 2.30910.0'
 
   if defined?($RNFirebaseAsStaticFramework)
     Pod::UI.puts "#{s.name}: Using overridden static_framework value of '#{$RNFirebaseAsStaticFramework}'"
