diff --git a/node_modules/@types/react-native-push-notification/index.d.ts b/node_modules/@types/react-native-push-notification/index.d.ts
index 1a36801..27c06fc 100644
--- a/node_modules/@types/react-native-push-notification/index.d.ts
+++ b/node_modules/@types/react-native-push-notification/index.d.ts
@@ -143,7 +143,7 @@ export interface PushNotification {
      * @deprecated use cancelLocalNotification method
      */
     cancelLocalNotifications(details: { id: string }): void;
-    cancelLocalNotification(notificationId: string): void;
+    cancelLocalNotification(notificationId: string | number): void;
     clearLocalNotification(tag: string, notificationID: number): void;
     cancelAllLocalNotifications(): void;
     setApplicationIconBadgeNumber(badgeCount: number): void;
