diff --git a/node_modules/react-native-wheel-pick/android/src/main/java/com/tron/ReactWheelCurvedPicker.java b/node_modules/react-native-wheel-pick/android/src/main/java/com/tron/ReactWheelCurvedPicker.java
index 60777b8..f6e52ba 100644
--- a/node_modules/react-native-wheel-pick/android/src/main/java/com/tron/ReactWheelCurvedPicker.java
+++ b/node_modules/react-native-wheel-pick/android/src/main/java/com/tron/ReactWheelCurvedPicker.java
@@ -15,7 +15,8 @@ import com.aigestudio.wheelpicker.WheelPicker.OnWheelChangeListener;
 import com.facebook.react.bridge.Arguments;
 import com.facebook.react.bridge.ReactContext;
 import com.facebook.react.bridge.WritableMap;
-import com.facebook.react.uimanager.UIManagerModule;
+import com.facebook.react.uimanager.UIManagerHelper;
+import com.facebook.react.uimanager.common.UIManagerType;
 import com.facebook.react.uimanager.events.Event;
 import com.facebook.react.uimanager.events.EventDispatcher;
 import com.facebook.react.uimanager.events.RCTEventEmitter;
@@ -35,7 +36,9 @@ public class ReactWheelCurvedPicker extends WheelPicker {
 
     public ReactWheelCurvedPicker(ReactContext reactContext) {
         super(reactContext);
-        mEventDispatcher = reactContext.getNativeModule(UIManagerModule.class).getEventDispatcher();
+
+
+        mEventDispatcher = UIManagerHelper.getUIManager(reactContext, UIManagerType.FABRIC).getEventDispatcher();
         setOnWheelChangeListener(new OnWheelChangeListener() {
             @Override
             public void onWheelScrolled(int offset) {
