diff --git a/node_modules/react-native-drawer/index.js b/node_modules/react-native-drawer/index.js
index ad60ecc..b949f00 100644
--- a/node_modules/react-native-drawer/index.js
+++ b/node_modules/react-native-drawer/index.js
@@ -117,7 +117,7 @@ export default class Drawer extends Component {
     this._childDrawer = drawer
   }
 
-  componentWillMount() {
+  UNSAFE_componentWillMount() {
     if (this.context.drawer) this.context.drawer._registerChildDrawer(this)
     if (this.props.openDrawerThreshold && process.env.NODE_ENV !== 'production') console.error('react-native-drawer: openDrawerThreshold is obsolete. Use panThreshold instead.')
     if (this.props.panStartCompensation && process.env.NODE_ENV !== 'production') console.error('react-native-drawer: panStartCompensation is deprecated.')
@@ -125,7 +125,7 @@ export default class Drawer extends Component {
     this.initialize(this.props)
   }
 
-  componentWillReceiveProps(nextProps) {
+  UNSAFE_componentWillReceiveProps(nextProps) {
     if (this.requiresResync(nextProps)) this.resync(null, nextProps)
 
     if (nextProps.open !== null && this._open !== nextProps.open) {
